modules/graphics/weston.ini.nix

```diff
@@ -34,6 +34,10 @@
       [launcher]
       path=${pkgs.gala-app}/bin/gala --enable-features=UseOzonePlatform --ozone-platform=wayland
       icon=${pkgs.weston}/share/weston/icon_terminal.png
+
+      [launcher]
```

Comment:
Could you please make this conditional? Only add this launcher icon if on aarch64-linux or NVIDIA

Reply:
This is a good point, will do.

Reply:
Ok, I made it conditional and rebased on top of main.